x-pack/plugins/cases/common/constants/index.ts

```diff
@@ -113,6 +113,9 @@ export const MAX_CATEGORY_FILTER_LENGTH = 100 as const;
 
 export const MAX_TITLE_LENGTH = 160 as const;
 export const MAX_CATEGORY_LENGTH = 50 as const;
+export const MAX_DESCRIPTION_LENGTH = 30000 as const;
+export const MAX_TAG_LENGTH = 256 as const;
+export const MAX_TAGS = 200 as const;
```

Comment:
I think in the end we should rethink all these names and make them consistent 😅 

I have been appending _LENGTH at the end of my constants when the type is an array but I noticed you do it only for strings where you wanna limit the length.

No need to do anything now but we should probably discuss it offline to have it consistent 😛

Reply:
yeah, I was also thinking about it. I wasn't sure `MAX_TAGS_LENGTH` will be clear enough that it's a length of an array.

Reply:
Maybe we can start being more explicit like `MAX_TAGS_PER_CASE`, etc